src/app/cube/donate/donate.component.html

```diff
@@ -2,25 +2,50 @@
   <section class="hero">
       <div class="inner">
           <h1 class="header">Donate To CLARK</h1>
-          <p class="description">CLARK is a digital library that hosts a diverse collection of free cybersecurity Learning Objects. In order to maintain our system and continue to provide our Learning Objects at no cost we need your help.</p>
+          <p class="description">The CLARK Platform is developed and maintained by TU students and relies on government and industry funding for support.</p>
```

Comment:
might wanna change the wordings a little, instead of "industry funding" to "industrial funding".

Reply:
everything else looks good

Reply:
Industrial funding actually refers to a specific fee for a tax form I believe.

Reply:
I've been spending a little too much time with accountants